uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -257,7 +256,7 @@ private void performMockMvcCallAndAssertResponse(MockHttpServletRequestBuilder m
             assertNotNull(actualConnectionDetails);
             assertEquals(expectedConnectionDetails.getSubdomain(), actualConnectionDetails.getSubdomain());
             assertEquals(expectedConnectionDetails.getUri(), actualConnectionDetails.getUri());
-            assertEquals(expectedConnectionDetails.getDashboardUri(), actualConnectionDetails.getDashboardUri());
+            assertNotNull(actualConnectionDetails.getDashboardUri());
```

Comment:
Could you at least assert that dashboard url in actualConnectionDetails contains a base dashboard url + "#/login"

Reply:
Done